pkgs/applications/audio/cider/default.nix

```diff
@@ -26,5 +30,6 @@ appimageTools.wrapType2 rec {
     license = licenses.agpl3;
     maintainers = [ maintainers.cigrainger ];
     platforms = [ "x86_64-linux" ];
+    mainProgram = "cider";
```

Comment:
Notice how `meta` attributes are sorted. They are _usually_ (not all the time) sorted alphabetically, can you please keep this order when adding `mainProgram` ?

Reply:
Oh well, will adjust 😆

Reply:
Should this be in some sort of contributing guideline?

Reply:
Not for that, it's just common good sense.

Reply:
Not a blocker. Also someone already done it, and the merge conflict should be solved.